galaxy_ng/app/common/proxy_url.py

```diff
@@ -0,0 +1,66 @@
+"""
```

Comment:
If this isn't used anywhere else, you're probably better off just putting these functions in the serializer.

Reply:
I think it is better to keep this as a separate module, because it is temporary, we will remove it anyway when pulp changes internally. Keeping those functions separated do not polute the serializer too much